common/dynamicconfig/configstore/config_store_client.go

```diff
@@ -133,15 +133,16 @@ func (csc *configStoreClient) startUpdate() error {
 	return nil
 }
 
-func (csc *configStoreClient) GetValue(name dc.Key, defaultValue interface{}) (interface{}, error) {
-	return csc.getValueWithFilters(name, nil, defaultValue)
+func (csc *configStoreClient) GetValue(name dc.Key) (interface{}, error) {
+	return csc.getValueWithFilters(name, nil, name.DefaultValue())
 }
 
-func (csc *configStoreClient) GetValueWithFilters(name dc.Key, filters map[dc.Filter]interface{}, defaultValue interface{}) (interface{}, error) {
-	return csc.getValueWithFilters(name, filters, defaultValue)
+func (csc *configStoreClient) GetValueWithFilters(name dc.Key, filters map[dc.Filter]interface{}) (interface{}, error) {
+	return csc.getValueWithFilters(name, filters, name.DefaultValue())
 }
 
-func (csc *configStoreClient) GetIntValue(name dc.Key, filters map[dc.Filter]interface{}, defaultValue int) (int, error) {
+func (csc *configStoreClient) GetIntValue(name dc.IntKey, filters map[dc.Filter]interface{}) (int, error) {
+	defaultValue := name.DefaultInt()
```

Comment:
fairly sure these should be `name.DefaultValue()` like the other cases (GetValue and earlier config comments)

Reply:
Nope, I was misinterpreting things.  this is good 👍